kolibri/plugins/learn/assets/src/state/actions/main.js

```diff
@@ -798,13 +667,12 @@ function closeExam(store) {
 }
 
 export {
+  _contentState,
```

Comment:
I don't think we have an established convention in the codebase, but typically underscores are associated with "private" or "protected" methods.

In the context of modules, of course anything that's not exported is private so the underscore is redundant or maybe just confusing...